app/logic/Abstract/Account.ts

```diff
@@ -110,6 +110,12 @@ export class Account extends Observable {
   /** Saves the config in this account to disk.
    * Does not save the contents, e.g. messages. */
   async save(): Promise<void> {
+    throw new AbstractFunction();
+  }
+
+  /** Saves the config in this account and its dependent accounts to disk. */
+  async saveAll(): Promise<void> {
```

Comment:
Good idea. In most cases when we save, we want to save only that one account, not the dependent accounts.